tests/testprogs/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ foreach(testprog_source ${testprog_sources})
       LINK_SEARCH_END_STATIC FALSE
       COMPILE_FLAGS "${testprog_cflags}"
       LINK_FLAGS "-no-pie")
+  if (${testprog_name} STREQUAL "inline_function")
```

Comment:
Just curious, is there a benefit of doing this rather than just marking square as `static inline`?

Reply:
I don't like this either 😕 How can it be better implemented?
If the function is `static inline`, the compiler will be able to always inline it and won't generate an out-of-line function. So the test won't verify that both inlined and regular call are probed.

Reply:
> So the test won't verify that both inlined and regular call are probed.

Don't we have other tests that already verify that we can probe non-inlined functions?

Reply:
Yeah... In that case I'll go with `static inline` and we can avoid this weird exception in the CMake!